server/wfnews-api-rest-endpoints/src/main/resources/application.properties

```diff
@@ -1,29 +1,31 @@
-wfone.datasource.url=#{wfone.datasource.url}
-wfone.datasource.username=#{wfone.datasource.username}
-wfone.datasource.max.connections=#{wfone.datasource.max.connections}
+wfone.datasource.url=jdbc:postgresql://wfone-public-mobile-db-nonprod.caw9xhtwiyin.ca-central-1.rds.amazonaws.com:5432/wfone_dev_pm
```

Comment:
Use configs, do not hardcode values
If you hardcode them locally, don't commit them